cli/src/commands/debug.rs

```diff
@@ -246,20 +253,43 @@ fn cmd_debug_watchman(
     command: &CommandHelper,
     subcommand: &DebugWatchmanSubcommand,
 ) -> Result<(), CommandError> {
+    use jj_lib::local_working_copy::LockedLocalWorkingCopy;
+
     let workspace_command = command.workspace_helper(ui)?;
     let repo = workspace_command.repo().clone();
     match subcommand {
         DebugWatchmanSubcommand::QueryClock => {
-            let (clock, _changed_files) = workspace_command.working_copy().query_watchman()?;
+            let Some(wc): Option<&LocalWorkingCopy> =
+                workspace_command.working_copy().as_any().downcast_ref()
+            else {
+                return Err(user_error(
+                    "This command requires a standard local-disk working copy",
+                ));
+            };
```

Comment:
nit: this looks a bit noisy, so how about adding a helper function?

Reply:
Done.